fourkitchens/suhumsci
# SHS-4986: Fix JS error in Gin admin theme

# READY FOR REVIEW

## Summary
- Add error handling to autocomplete resize js code (fixes [SHS-4986](https://app.clickup.com/t/36718269/SHS-4986))

## Steps to Test
1. Edit the site settings file (`docroot/sites/[SITE]/settings/local.settings.php`) and enable CSS and JS aggregation (disabled by default):

```
$config['system.performance']['css']['preprocess'] = TRUE;
$config['system.performance']['js']['preprocess'] = TRUE;
```

2. Clear cache
3. Visit any admin page using the Gin theme (for example `/admin/structure/menu/manage/main`) and confirm that all JS code (toolbar, sorting, actions menu, etc.) works as expected, without any error in the browsers console
4. Masquerade or login as an user with the contrib role. Edit the site homepage and confirm that you can edit any paragraph, without any error.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
